benchmarks/benchmarks/bench_linalg.py

```diff
@@ -2,7 +2,6 @@
 
 import numpy as np
 
-
```

Comment:
[PEP 8](https://www.python.org/dev/peps/pep-0008/#blank-lines) recommends two blank lines before top-level classes and functions.